### ISSUE TITLE
Feature request: Option for address label to follow change addresses

### ISSUE THREAD

**Issue by viperperidot** [spesmilo/electrum#6652]
Not sure if feature requests are accepted on here but if not then I can delete this post.

I am wondering if it would be possible to allow an option for address labels to continue on to associated change addresses.

So for example: I have a label on address A, I spend an amount that does not equal the total from that address, the label I had on address A now appears on address B (the change output from the transaction).

This is useful for people who practice coin control and want their label to follow utxos until they are completely spent.

Maybe you could enable this option by right clicking on an address in the Addresses tab and selecting something because of course some people will not want this feature enabled all the time for all addresses.

**Comment by ecdsa** [comment]
yes, I think this would be useful. note that Wasabi has a similar feature.
